src/models/user.js

```diff
@@ -34,8 +34,10 @@ export default (sequelize, DataTypes) => {
     roleId: {
       type: DataTypes.INTEGER,
       allowNull: false,
-    },
+    }
   });
-
+  User.associate = models => {
+    User.hasOne(models.TripRequest, { foreignKey: 'userId', onDelete: 'CASCADE' });
```

Comment:
This association is invalid, a user should be able to make multiple trip request over time. It should be `User.hasMany(models.TripRequest...`

Reply:
noted